cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from cfme.common.provider import cleanup_vm
+from cfme.infrastructure.provider import InfraProvider
+from cfme.services.catalogs.service_catalogs import ServiceCatalogs
+from cfme.services.requests import Request
+from cfme.services.dashboard import Dashboard
+from cfme import test_requirements
+from utils.log import logger
+from utils import testgen
+from utils.version import current_version
+from utils.appliance import get_or_create_current_appliance, ViaSSUI
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('vm_name', 'catalog_item', 'uses_infra_providers'),
+    test_requirements.ssui,
+    pytest.mark.long_running
+]
+
+
+pytest_generate_tests = testgen.generate([InfraProvider], required_fields=[
+    ['provisioning', 'template'],
+    ['provisioning', 'host'],
+    ['provisioning', 'datastore']
+], scope="module")
+
+
+@pytest.fixture
+def order_catalog_item_in_ops_ui(provider, setup_provider, catalog_item, request):
+    """Tests order catalog item
+    Metadata:
+        test_flag: provision
+    """
```

Comment:
Regarding docstrings, I see that some classes are still undocumented and some docstring doesn't reflect reality - e.g. this one clearly belongs to test.
Also would be nice if all docstrings followed the same conventions of google docstrings that we use - e.g. http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html